### PR TITLE
Sawja.1.5.7

### PR DESCRIPTION
**Pull request by anscoil**
New Sawja release removing `camlp4` dependency.

### CODE REVIEW

File: packages/sawja/sawja.1.5.7/opam [hunk 1]
```diff
@@ -0,0 +1,33 @@
+opam-version: "2.0"
+maintainer: "sawja@inria.fr"
+build: [
+  ["./configure.sh"]
+  [make]
+]
+remove: [
```
Comment (kit-ty-kate) [inline packages/sawja/sawja.1.5.7/opam]: The `remove section is unnecessary since opam 2.0

File: packages/sawja/sawja.1.5.7/opam [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+authors: "Sawja development team"
+install: [
+  [make "install"]
+]
+homepage: "http://sawja.inria.fr"
+depends: [
+  "ocaml" {>= "4.02"}
+  "ocamlfind" {build}
+  "javalib" {>= "3.1.1" & < "3.2"}
+]
```
Comment (kit-ty-kate) [inline packages/sawja/sawja.1.5.7/opam]:
```suggestion
  "conf-perl" {build}
]
```
Seems like your package also requires `conf-perl`:
```
#=== ERROR while compiling sawja.1.5.7 ========================================#
# context              2.0.5 | linux/x86_64 | ocaml-base-compiler.4.08.1 | pinned(https://gforge.inria.fr/frs/download.php/file/38117/sawja-1.5.7.tar.bz2)
# path                 ~/.opam/4.08/.opam-switch/build/sawja.1.5.7
# command              ~/.opam/4.08/.opam-switch/build/sawja.1.5.7/./configure.sh
# exit-code            83
# env-file             ~/.opam/log/sawja-13-dc6b9b.env
# output-file          ~/.opam/log/sawja-13-dc6b9b.out
### output ###
# * Debug flag set to yes.
# * Buddy flag set to no. Sawja won't use the buddy BDD package.
# * Ocamlfind found at /home/opam/.opam/4.08/bin/ocamlfind.
# * System-wide installation, in /home/opam/.opam/4.08/lib.
# 
# ! configure error: Perl not found.
# 
# Use your system's software packaging tools to install Perl, or download it from:
# http://www.perl.org/get.html.
```

**Comment by kit-ty-kate** [comment]
Thanks a lot

**Comment by anscoil** [comment]
Thanks to you

**Pull request merged**